src/screens/activity/progress/reflection-of-the-day/ReflectionOfTheDay.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useEffect } from 'react'
+import { Text, View, Alert, ScrollView, KeyboardAvoidingView, StatusBar } from 'react-native'
+import { useNavigation } from '@react-navigation/native'
+import * as Progress from 'react-native-progress'
+
+import ASHeader from '../../../../components/header/ASHeader'
+import ASQuestionCard from '../../../../components/question-card/ASQuestionCard'
+import { BottomTabNavigationProp } from '@react-navigation/bottom-tabs'
+import { COLORS } from '../../../../theme/colors'
+import { Spacing } from '../../../../theme/spacing'
+import { TabNavigatorParams } from '../../../../types/tabNavigator-types'
+
+import { QUESTIONS_DATA } from '../../../../constants/activity-constants'
+import { whiteBackArrowIcon } from '../../../../constants/common-constants'
+
+import { styles } from './relectionOfTheDay-styles'
+
+const ReflectionOfTheDay = () => {
+  const [answers, setAnswers] = useState<string[]>([])
+  const [currentQuestionNumber, setCurrentQuestionNumber] = useState<number>(0)
+  const [input, setInput] = useState<string>('')
```

Comment:
can also be written as:
const [input, setInput] = useState('')

type cast is not required at every place